src/components/media-views.js

```diff
@@ -545,26 +549,39 @@ AFRAME.registerComponent("media-video", {
   },
 
   tick() {
+    if (!this.video) return;
+
     const userinput = this.el.sceneEl.systems.userinput;
     const volumeMod = userinput.get(paths.actions.cursor.mediaVolumeMod);
-    if (volumeMod) {
+    if (this.el.is("hovered") && volumeMod) {
       this.el.setAttribute("media-video", "volume", THREE.Math.clamp(this.data.volume + volumeMod, 0, 1));
+      this.volumeLabel.setAttribute(
```

Comment:
You might avoid some work with:
```
      this.volumeLabel.components.text.updateGeometry(this.geometry, font);
      this.volumeLabel.components.text.updateLayout();
```
We have reservations for `setAttribute` and also the `update` for the `text` component does some checks that aren't necessary for you (because you aren't changing font or shader or texture here.

Reply:
This would leave the components data in a weird state. Since this is only when adjusting volume I am not too worried about it.